tools/metabase/openshift/metabase.dc.yaml

```diff
@@ -189,34 +188,12 @@ parameters:
     description: Maximum amount of memory allowed to use
     displayName: Memory Limit
     value: 1.5Gi
-  - name: PVC_SIZE
-    description: Amount of disk space needed for persistence
-    displayName: PVC Size
-    required: true
-    value: 2Gi
-  - name: ADMIN_EMAIL
-    displayName: Admin Email
-    description: The email associated with the metabase admin user
-    required: true
-  - name: ADMIN_PASSWORD
-    displayName: Admin Password
-    description:
-      The password for the metabase admin user. Requires 2 upper, 2 lower,
-      1 special, 1 numeric chars and minimum 8 char length
-    generate: expression
-    from: "[a-zA-Z0-9]{8}!@#$%^&_[a-zA-Z0-9]{8}"
-    required: true
-  - name: DATABASE_PASSWORD
-    displayName: Database Password
-    description:
-      A minimum 16 character password that is generated in the target database,
-      and then copied over into this field.
-    generate: expression
-    from: "[a-zA-Z0-9]{16}"
-    required: true
-  - name: EMBEDDING_SECRET_KEY
-    displayName: Embedding Secret Key
-    description: A key that is used to embed metabase screens from within a website
-    generate: expression
-    from: "[a-zA-Z0-9]{16}"
+  - name: DATA_VOLUME_CAPACITY
+    description: Volume space available for Data
+    displayName: Data Volume Capacity (use M or G, not Mi or Gi)
     required: true
+    value: 1Gi
```

Comment:
says don't use Gi in line above (?)

Reply:
Probably ancient OCP3 specific note. It needed to be Gi for it to register. Removing that note.